packages/launchpad/cypress/tasks/snapshotsScaffold.ts

```diff
@@ -40,9 +40,9 @@ async function snapshotScaffoldedFiles (expectedScaffoldDir: string, filesToDiff
 function compareScaffoldedFiles (filesToDiff: FileToDiff[]): SnapshotScaffoldTestResult {
   for (const { actual, expected } of filesToDiff) {
     try {
-      const read = (f: string) => fs.readFileSync(f, 'utf-8')
-      const actualContent = read(actual).trim()
-      const expectedContent = read(expected).trim()
+      const read = (f: string) => fs.readFileSync(f, 'utf-8').trim().replaceAll('\r\n', '\n')
```

Comment:
windows line-endings was messing with the diff. I wish (wonder? maybe there is?) if there's a more idiomatic way that this. I wish `fs.readFileSync(..., { normalizeLineEndings: "\n")` or something.